docs/specification.md

```diff
@@ -544,9 +537,9 @@ unit
 
 ```haskell
 data Proposal proposalMetadata = Proposal
-    { frozen_token_requirement :: Natural
-    -- ^ How many tokens should be frozen in order to vote on this proposal.
-    , metadata :: proposalMetadata
+    { metadata :: proposalMetadata
+    , proposer :: Address
+    , proposerFrozenToken :: Natural
```

Comment:
Now it looks confusing. There is also `Natural` inside `Propose` constructor. How are these 2 different? AFAIU only one should be present.

Reply:
> Apart from that the proposer specifies how many tokens one should lock in order to vote on this proposal.

I've remove this.

> Now it looks confusing. There is also Natural inside Propose constructor. How are these 2 different? AFAIU only one should be present.

I've updated this to use `ProposeParam` instead.